### DISCUSSION TITLE
Deepgram transcription service substantially deteriorated in accuracy to almost unusable.

### DISCUSSION THREAD

**Discussion by cowanAI** [deepgram/community#344] [General help] [answered]
### Which Deepgram product are you using?

Deepgram SDKs

### Details

Im using Deepgram for my medical app but recently the performance is nearly unusable, I used even the official website demo because I thought I was using the SDK incorrectly but this was not the case:

<img width="1331" alt="image" src="https://github.com/deepgram/community/assets/128092052/fe74d804-971d-4edc-8fb8-43332b7d2aa7">

### If possible, please attach your code or paste it into the text box.

You can use the website demo for this: 
<img width="1331" alt="image" src="https://github.com/deepgram/community/assets/128092052/7648a149-dc13-42f4-9eca-cd468d33b25b">

### If possible, please attach an example audio file to reproduce the issue.

<img width="1331" alt="image" src="https://github.com/deepgram/community/assets/128092052/f33fdb41-2120-4c62-9798-e3d37313bcc4">

**Comment by cowanAI** [comment]
Also, here's the live demo recorded using a professional voice over actor with 0 accent so the audio quality is not an issue as well: https://drive.google.com/file/d/1L7DCQgxdpvml-ZPAGPJ2zcs6OhSBpeoN/view?usp=sharing

**Reply by lukeocodes** [reply]
Did you share the wrong file?

**Reply by cowanAI** [reply]
yes, sorry about that, I have updated to the correct one.

**Comment by jtoh89** [comment]
Just started using it, but I've been seeing poor transcription with industry related vocabulary. Seems like its similar to your issue. Have you looked into custom models through Deepgram?

**Reply by cowanAI** [reply]
Ohh custom models? I went through the documentation but could not find any, I would incredibly appreciate if you had the link in the meantime searching.

**Reply by cowanAI** [reply]
ohh I see now, you have to contact them, tried doing that but never received an email back, had to switch to azure for emergency, it would be great if they published the name of a custom medical model because the results were just very poor.

**Reply by jtoh89** [reply]
@cowanAI try reaching out again, I got an email response within couple hrs. Are you seeing better accuracy with azure?

**Reply by cowanAI** [reply]
like 10 times more accurate, no joke, just use the audio from my video and use both at the same time. Don't know what happened to deepgram, still waiting for their team to get back to me.

**Reply by scottstephenson** [reply, marked as answer]
You should ask for access to the medical model and see how it works for you. You can give feedback to the product team about it too. Right now you are using the general model which isn't going to do all that well on medical terminology.

This is one way to let the team know what beta features/models you are interested in:
https://github.com/orgs/deepgram/discussions/333

You can share direct feedback in the console view too.

Just click Share feedback in the lower left corner:
<img width="161" alt="image" src="https://github.com/deepgram/community/assets/19791155/8d791682-348c-47d5-b862-8c24c8c94db5">

And let 'em know what you think:
<img width="753" alt="image" src="https://github.com/deepgram/community/assets/19791155/adc48b8c-28f3-4eb1-bc44-264f9af57398">

You can email like was suggested before by @jtoh89 too:
<redacted>

**Reply by cowanAI** [reply]
Thanks so much!! this was the answer I was hoping! I will email today to <redacted> for the medical model, it is my intention to give you guys all my money!

**Reply by cowanAI** [reply]
Just want to say to everyone that we tried the new NOVA V2 model and it hit the ball out of the park, thanks so much deepgram team!!

**Reply by jtoh89** [reply]
Awesome, how did you get access? I just reached out to their support. Crossing my fingers!

**Reply by cowanAI** [reply]
checked their request data on their playground and used that model name on my app and it worked great.
